pkgs/development/arduino/arduino-core/default.nix

```diff
@@ -84,7 +84,7 @@ stdenv.mkDerivation rec {
     url = "http://downloads.arduino.cc/arduino-${version}-${teensy_architecture}.tar.xz";
     sha256 =
       lib.optionalString ("${teensy_architecture}" == "linux64")
-        "1f8s3by5lc6fazyaa9zc9kz3ar8zj8jabab1fy5jzh49fbd8bydx"
+        "1lv4in9j0r8s0cis4zdvbk2637vlj12w69wdxgcxcrwvkcdahkpa"
       + lib.optionalString ("${teensy_architecture}" == "linux32")
         "1r9ral9aq5vp02dwgagifk5h403l7knxdyi1w23rqpcbbpa423lw"
```

Comment:
the other architectures should be updated as well, otherwise they will be broken.